doc/articles/uno-build-error-codes.md

```diff
@@ -0,0 +1,20 @@
+---
+uid: Build.Solution.error-codes
+---
+# Uno Build error codes
+
+## UNOB0001: Cannot build with both Uno.WinUI and Uno.UI NuGet packages referenced
+
+This error code means that a project has determined what both `Uno.WinUI` and `Uno.UI` packages are referenced.
+
+To fix this issue, you may be explicitly referencing `Uno.UI` and `Uno.WinUI` in your `csproj`, or you may be referencing a NuGet package that is incompatible with your current project's configuration.
+
+For instance, if your project references `Uno.WinUI`, and you try to reference `SkiaSharp.View.Uno`, you will get this error. To fix it, you'll need to reference `SkiaSharp.View.Uno.WinUI` instead.
+
+## UNOB0002: Project XX contains a reference to Uno Platform but does not contain a WinAppSDK compatible target framework.
```

Comment:
does https://aka.platform.uno/UNOB0002 redirect here somehow?

Reply:
It will be redirecting to that exact spot, once the docs are published :)